generator/beat/{beat}/magefile.go

```diff
@@ -0,0 +1,73 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// +build mage
+
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/magefile/mage/mg"
+
+	devtools "github.com/elastic/beats/dev-tools/mage"
+
+	// mage:import
+	_ "github.com/elastic/beats/dev-tools/mage/target/common"
+	// mage:import
+	"github.com/elastic/beats/dev-tools/mage/target/build"
+	// mage:import
+	"github.com/elastic/beats/dev-tools/mage/target/update"
+	// mage:import
+	_ "github.com/elastic/beats/dev-tools/mage/target/test"
+	// mage:import
+	_ "github.com/elastic/beats/dev-tools/mage/target/unittest"
+	// mage:import
+	"github.com/elastic/beats/dev-tools/mage/target/pkg"
+)
+
+func init() {
+	devtools.SetBuildVariableSources(devtools.DefaultBeatBuildVariableSources)
+
+	devtools.BeatDescription = "One sentence description of the Beat."
+	devtools.BeatVendor = "{full_name}"
```

Comment:
This line was the reason to templatize this file, do we have another way of replacing it?

Reply:
Context for the change in #13195

Reply:
It still works. Changing the `.tmpl` is a no-op. I just found the extension irritating. The make files are technically 'templates' but they don't actually have that extension.

Reply:
Oh ok, it is a bit confusing to have the tmpl files, maybe we should remove the extension from the rest of go files too, wdyt?